resources/config/laravel-sitemap.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
We don't need a `resources` dir. Name this config file `sitemap.php` and put it an a top level `config` directory. 

Be sure to update things wherever your read the config file.